src/tokens/selectors.ts

```diff
@@ -187,7 +187,7 @@ export const tokensByUsdBalanceSelector = createSelector(
  * @deprecated
  */
 export const coreTokensSelector = createSelector(tokensByUsdBalanceSelector, (tokens) => {
```

Comment:
I didn't bother to rename this selector to `feeCurrencyTokensSelector` according to the change, since it's deprecated